src/graphql/lib/MagentoArguments.js

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
```

Comment:
If this is to map cif args to magento can you rename it to suggest this? like MagentoArgsMapper or CifToMagentoArgsMapper?